_targets.R

```diff
@@ -0,0 +1,38 @@
+library(targets)
+source("1_fetch/src/fetch_sb.R")
+source("2_process/src/prep_data.R")
+source("3_visualize/src/plot_data.R")
+tar_option_set(packages = c("tidyverse", "stringr", "sbtools", "whisker"))
+
+list(
+  # Get the data from ScienceBase
+  tar_target(
+    model_RMSEs_csv,
+    fetch_sb(fetch_dir = "1_fetch/out/",sb_file = "me_RMSE.csv", sbitem = "5d925066e4b0c4f70d0d0599", out_file = "model_RMSEs.csv"),
+    format = "file"
+  ), 
+  # Prepare the data for plotting
+  tar_target(
+    eval_data,
+    prep_data(file_in = model_RMSEs_csv)
+  ),
+  # Create a plot
+  tar_target(
+    figure_1_png,
+    plot_data(plot_out = "3_visualize/out/figure_1.png", eval_data), 
+    format = "file"
+  ),
+  # Save the processed data
+  tar_target(
+    model_summary_results_csv,
+    write_rmse(eval_data, file_out = "2_process/out/model_summary_results.csv"), 
```

Comment:
I can't remember if the course has talked about this yet (so potentially _**spoiler alert**_) but one way to get around having to write a custom function to make sure that the target command returns the filepath for file targets is to use `{ }` and put your code directly in the targets makefile. Not something you want to do when there are lots of lines of code, but nice when it is something simple, such as needing to return the filepath.

E.g.
```r
tar_target(
  model_summary_results_csv,
  {
    file_out <- "2_process/out/model_summary_results.csv"
    write_csv(eval_data, file = file_out)
    return(file_out)
  }, 
  format = "file"
),
```